airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -137,6 +138,9 @@ public class ConnectionManagerWorkflowImpl implements ConnectionManagerWorkflow
   @Trace(operationName = WORKFLOW_TRACE_OPERATION_NAME)
   @Override
   public void run(final ConnectionUpdaterInput connectionUpdaterInput) throws RetryableException {
+    if (connectionId == null) {
```

Comment:
this will always be null. I think that @jdpgrailsdev is about to do a similar change

Reply:
See https://github.com/airbytehq/airbyte-platform-internal/pull/6277